Microsoft.NET.Build.Containers/build/Microsoft.NET.Build.Containers.targets

```diff
@@ -1,4 +1,18 @@
 <Project>
+    <Target Name="_ContainerVerifySDKVersion"
+        Condition="'$(WebPublishMethod)' == 'Container' or '$(PublishProfile)' == 'DefaultContainer'"
+        BeforeTargets="AfterPublish">
+        <!-- If the user has opted into container publishing via their own profile (WebPublishMethod = Container) or
+            via the default Profile (PublishProfile = DefaultContainer), make sure they're on a supported SDK version.
+            We do the explicit profile name check here because for preview6 for example the profile didn't exist, so we
+            can't rely only on the WebPublishMethod. -->
+        <PropertyGroup>
+            <!-- Allow preview 7, any RC, or any stable version of 7 -->
+            <_IsAllowedVersion Condition="$(NETCoreSdkVersion.StartsWith('7.0.100-preview.7')) or $(NETCoreSdkVersion.StartsWith('7.0.100-rc')) or ($(NETCoreSdkVersion.StartsWith('7.0.10')) and $(NETCoreSdkVersion.Contains('-')) == false)">true</_IsAllowedVersion>
```

Comment:
The `($(NETCoreSdkVersion.StartsWith('7.0.10'))` is a superset of the prior two.  technically allow some earlier preview than 7?

Reply:
nevermind, I misread the parentheses.